src/main/java/uk/gov/justice/hmpps/prison/service/receiveandtransfer/ExternalMovementTransferService.kt

```diff
@@ -144,6 +145,86 @@ class ExternalMovementTransferService(
       )
     }
   }
+  fun updateMovementsForTransferInAfterTemporaryAbsenceToSamePrison(
```

Comment:
TODO plan to refactor this class, basically most of the methods do roughly the same thing